style.css

```diff
@@ -333,3 +345,256 @@ about me section
   -ms-border-radius: 8px;
   -o-border-radius: 8px;
 }
+
+/* =====================
+contact me section
+===================== */
+.contactme {
+  text-align: center;
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+.contactme h2 {
+  margin-top: 60px;
+  font-size: 40px;
+  line-height: 52px;
+  font-weight: 700;
+  color: #172b4d;
+  margin-bottom: 58px;
+}
+
+.contactme form {
+  display: flex;
+  flex-direction: column;
+}
+
+/* add styling to form elements -ram */
+.form-element {
+  height: 48px;
+  border: 2px solid #d0d9d4;
+  color: #6b778c;
+  border-radius: 4px;
+  margin-bottom: 24px;
+  font-size: 15px;
+  line-height: 24px;
+  text-align: left;
+  padding: 15px;
+}
+
+.msg-text {
+  height: 30%;
+}
+
+.form-button {
+  margin: 0 auto;
+}
+
+.social-footer {
+  margin-top: 10vh;
+  display: flex;
+  flex-direction: column;
+  gap: 3vh;
+  align-items: center;
+}
+
+.postal-email-id {
+  font-family: "Inter", sans-serif;
+  font-size: 2.5rem;
+  margin-bottom: 189px;
+}
+
+.social-footer hr {
+  margin-top: 134px;
+  margin-bottom: 20px;
+  border-top: 4px solid black;
+  width: 30%;
+  display: inline-flex;
+  align-items: center;
+}
+
+.firstlastname {
+  display: none;
+}
+
+.dividerfoo {
+  display: none;
+}
+
+@media screen and (min-width: 800px) {
```

Comment:
- [ ] Well done :+1:  on selecting minimum screen size. But, There is an issue. In the below `gif` you can see that, your design works fine with `1496px` means a very big screen, but when decreased the size, the design and alignment is collapsed. Kindly fixed this issue.

![ezgif com-gif-maker (2)](https://user-images.githubusercontent.com/60083437/206454679-1b966d42-3428-48ca-97d3-c6f587b326cf.gif)